app/models/user.rb

```diff
@@ -0,0 +1,19 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+
```

Comment:
![image](https://github.com/Munyabelden/Blog_App/assets/93039359/10d80ddb-3a4d-46c4-83b2-46f2003d3278)

- [ ] It would be nice if you follow the ERD diagram in which it is clear that the foreign key which comments and likes tables have is user_id not the author_id so it would be nice if you made your comments and likes table association as you can see below :arrow_down:

```RUBY
  has_many :posts, foreign_key: 'author_id'
  has_many :comments, foreign_key: 'user_id'
  has_many :likes, foreign_key: 'user_id'
```